definitions/base_classes/NXattenuator.nxdl.xml

```diff
@@ -88,4 +88,30 @@
             for a summary of the discussion.
         </doc>
     </attribute>
+    <field name="depends_on" type="NX_CHAR">
+      <doc>
+        Points to the path to a field defining the last transformation in a chain of transformations
+        which defines the position and orientation of this component or the string "." if located
+        in the origin.
+
+        The reference point of the attenuator is its center in the x and y axis. The reference point on the z axis is the
+        surface of the attenuator pointing towards the source.
+
+        For complex geometries, where identifing a centre in x and y is not obvious 
```

Comment:
centre or center - pick one or the other ;)

Reply:
I think we belong to the Data Management and Software Centre? :D

Reply:
I want to make a note that the original text is inconsistent as well. If I am to change/fix this I would simply do a search and replace on the whole directory.